packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.routes.js

```diff
@@ -0,0 +1,29 @@
+import template from './ipv4.html';
+import controller from './ipv4.controller';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.ip.agora-order.ipv4', {
+    url: '/ipv4',
+    views: {
+      '': {
+        template,
+        controller,
+        controllerAs: '$ctrl',
+      },
+    },
```

Comment:
```suggestion
    component
```

Use component file and inject template, controller and controllerAs as parameters of this component